polkadot/node/core/pvf/prepare-worker/benches/prepare_rococo_runtime.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use criterion::{criterion_group, criterion_main, Criterion, SamplingMode};
+use polkadot_node_core_pvf_common::{
+	executor_intf::{prepare, prevalidate},
+	prepare::PrepareJobKind,
+	pvf::PvfPrepData,
+};
+use polkadot_primitives::ExecutorParams;
+use std::time::Duration;
+
+fn do_prepare_runtime(pvf: PvfPrepData) {
+	let blob = match prevalidate(&pvf.code()) {
+		Err(err) => panic!("{:?}", err),
+		Ok(b) => b,
+	};
+
+	match prepare(blob, &pvf.executor_params()) {
+		Ok(_) => (),
+		Err(err) => panic!("{:?}", err),
+	}
+}
+
+fn prepare_rococo_runtime(c: &mut Criterion) {
+	let blob = rococo_runtime::WASM_BINARY.unwrap();
```

Comment:
nit: Use `rococo_parachain_runtime` instead?

Reply:
But it's not a parachain runtime, it's Rococo runtime itself :slightly_smiling_face:

Reply:
I mean, we're preparing something that doesn't export `validate_block` and thus can't be executed. It probably doesn't matter for the purpose of the benchmark, but could potentially be a point of confusion.

Reply:
Yes, that's right. But for preparation benchmarking, it's better to use code similar to a real-life parachain code by its volume, and there's nothing suitable at hand. All the test parachains are too simple. So, as we're not going to execute it anyway, I decided to use Rococo.

Reply:
Yeah, makes sense. On the other hand, the bench could break if we ever switch to your custom compiler. FWIU, it will either only support compiling PVFs, or at least be optimized specifically for them. Something to keep in mind at least.